### DISCUSSION TITLE
Missing projectId query parameter

### DISCUSSION THREAD

**Discussion by jacky-yanglei** [WalletConnect/walletconnect-monorepo#3404] [Developer Support]
**Describe the bug**
Last week, this code could be used normally. This week, it was found that there was an error reporting that there was no projectid parameter, which was passed in in the code.

**SDK Version (if relevant)**
 - Client: [JS]
 - Version [2.6.0]

**To Reproduce**

**Expected behavior**

**Screenshots**
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/17645850/ba20c1e6-082a-43ec-983b-4b2962e3047a)
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/17645850/5df4c4fd-8f22-46a0-bd3c-6184ee22a574)
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/17645850/8efb47f4-3387-4202-9d45-ace111300d33)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by FurkanSezal** [comment]
I have the same issue

**Comment by jpfraneto** [comment]
I have the same issue and I've been hitting my head against the wall all the morning trying to fix it.

**Comment by Fantasi-Dick** [comment]
I have same issue

**Comment by Fantasi-Dick** [comment]
> I have the same issue and I've been hitting my head against the wall all the morning trying to fix it.

I fix this bug just now. 
The reason for this bug was that both "wagmi" and this library were used simultaneously. After removing the "wagmi" related code, the issue was resolved.

**Comment by FurkanSezal** [comment]
> 

I use only wagmi though still same issue

**Comment by glitch-txs** [comment]
Hi could you provide a minimal reproducible example?

**Comment by FurkanSezal** [comment]
> 

After i saw this https://github.com/WalletConnect/walletconnect-monorepo/issues/3193#issuecomment-1645126974
i use a useState hook for wagmi stuff now its fixed.

**Comment by DragonmasterETH** [comment]
This happens with us as well on ethers but only on an external site, even when we whitelist the site and subdomain.
On localhost it works fine.

Will get a MRE up tomorrow night to hopefully solve this

**Comment by kalucki** [comment]
any updates on that? I'm getting this issue year after
